src/scss/grommet-core/_objects.form-field.scss

```diff
@@ -95,6 +95,11 @@ $colored-select-drop-caret: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAB
     @include form-field-content();
   }
 
+  > .#{$grommet-namespace}calendar input {
```

Comment:
why only for Calendar? shouldn't we add the same for DateTime and search-input ?

> .#{$grommet-namespace}search-input input,
  > .#{$grommet-namespace}calendar input,
  > .#{$grommet-namespace}date-time input {